personalize.admin.inc

```diff
@@ -85,6 +85,89 @@ function personalize_admin_form_submit($form, &$form_state) {
 }
 
 /**
+ * Form for changing the status of an agent.
```

Comment:
So with the toggle stuff there's an assumption of only one possible next status. I'm leaving it in place for now as the button on the actual campaign edit screen still uses it. I'll put in a todo about revisiting this though so we don't have all these different functions for essentially the same thing.